docs/config.md

```diff
@@ -1862,16 +1936,160 @@ connections this setting can generate.
 
 > [!CAUTION]
 > For nodes without strict connection limits that need to provide large volumes
-> of content immediately, we recommend enabling the `Routing.AcceleratedDHTClient` and
-> setting `Provider.WorkerCount` to `0` (unlimited).
+> of content, we recommend first trying `Provide.DHT.SweepEnabled=true` for efficient
+> announcements. If announcements are still not fast enough, enable
+> `Routing.AcceleratedDHTClient=true` and set `Provide.DHT.MaxWorkers` to `0` (unlimited).
```

Comment:
@guillaumemichel in the past we had this thing where people who wanted to announce new CIDs as fast as possible set `Provide.WorkerCount` to `0` (unlimited).

After this PR, this advice looks like this, but is still sensible to remove cap?

Perhaps we should reject `0` as value and instead ask people to just adjust based on their needs:


```suggestion
> announcements. If announcements are still not fast enough, enable
> `Routing.AcceleratedDHTClient=true` and adjust `Provide.DHT.MaxWorkers` and `DedicatedBurstWorkers`.
```